src/components/UserTable/UserTable.jsx

```diff
@@ -24,71 +26,127 @@ import AUTH_ROLES from '../../utils/AuthConfig';
 
 const { SUPERADMIN_ROLE, ADMIN_ROLE } = AUTH_ROLES.AUTH_ROLES;
 
-const UserTable = ({ isSuperAdmin, users }) => {
+const UserTable = ({
+  isSuperAdmin,
+  users,
+  setAllUsers,
+  setDriverUsers,
+  setAdminUsers,
+  updateDisplay,
+}) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { isOpen: deleteIsOpen, onOpen: deleteOnOpen, onClose: deleteOnClose } = useDisclosure();
+  const [editUserData, setEditUserData] = useState([]);
+  const [deleteUserData, setDeleteUserData] = useState([]);
```

Comment:
The default value for these should be empty objects (`{ }`) instead of empty lists, since the values they end up being set to are objects.